bugbug/model_calibration.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from sklearn.isotonic import IsotonicRegression
+from sklearn.model_selection import train_test_split
+
+
+# Wrapper class to calibrate a model
+class IsotonicRegressionCalibrator:
+    def __init__(self, model):
+        self.model = model
+        self.calibrated = False
+        self.X_train, self.X_val, self.X_test = None, None, None
+        self.y_train, self.y_val, self.y_test = None, None, None
+        self.ir = None
+
+    def split_data(self, X, y):
```

Comment:
so where should I split the data?

Reply:
> This is not called anywhere.

This code is not used.

> why do we need to store the datasets in the class itself?

Why do we need to store the datasets since they will be used just once to fit?

Reply:
split_data() is called in the train method!